src/components/form/results.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./form.scss";
+import { ListHeader } from "../list-header/list-header";
+
+function Results({ question, option }) {
+  // TODO: use a real component for this
+  return (
+    <div>
+      <ListHeader title={question.questionText} />
+      <div>
+        <div className={"__list-item no-hover"}>
+          <div className="__list-item__left-side" />
+          <div className="__list-item__content">{option.label}</div>
+        </div>
+      </div>
+    </div>
+  );
+}
```

Comment:
We have now a component for this. We can remove this one. Please use [this](https://taskforce-fe-compone-git-fork-mmircea16-form-with-decisi-2be921.code4ro.now.sh/?path=/story/listitem--with-non-option-state) one

Reply:
Also, please check [this](https://github.com/code4romania/ce-ma-fac/pull/164/files) PR for inspiration. There's been some nice updates for this part of the component.